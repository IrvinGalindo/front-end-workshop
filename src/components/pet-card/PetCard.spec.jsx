import React from 'react';
import { shallow } from 'enzyme';

import PetCard from './PetCard';

describe('PetCard', () => {
  const petCard = shallow(
    <PetCard id="abc-def" img="benito.png" name="Benito" details="some details about benito el perrito" />
  );

  it('should render', () => {
    expect(petCard).toMatchSnapshot();
  });
});
