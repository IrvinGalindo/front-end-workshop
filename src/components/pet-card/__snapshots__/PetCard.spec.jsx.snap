// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PetCard should render 1`] = `
<article
  className="yta_PetCard"
>
  <div
    className="yta_PetCard-img"
    style={
      Object {
        "backgroundImage": "url(benito.png)",
      }
    }
  />
  <div
    className="yta_PetCard-description"
  >
    <h3
      className="yta_PetCard-description-name"
    >
      Benito
    </h3>
    <p
      className="yta_PetCard-description-details"
    >
      some details about benito el perrito
    </p>
    <p
      className="yta_PetCard-description-link"
    >
      <a
        className="yta_Link-outline--primary"
        href="/pet/abc-def"
      >
        Adoptar
      </a>
    </p>
  </div>
</article>
`;
